group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/UniformAssignor.java

```diff
@@ -66,21 +66,19 @@ public GroupAssignment assign(
         GroupSpec groupSpec,
         SubscribedTopicDescriber subscribedTopicDescriber
     ) throws PartitionAssignorException {
-        AbstractUniformAssignmentBuilder assignmentBuilder;
-
         if (groupSpec.members().isEmpty())
             return new GroupAssignment(Collections.emptyMap());
 
         if (groupSpec.subscriptionType().equals(HOMOGENEOUS)) {
             LOG.debug("Detected that all members are subscribed to the same set of topics, invoking the "
                 + "optimized assignment algorithm");
-            assignmentBuilder = new OptimizedUniformAssignmentBuilder(groupSpec, subscribedTopicDescriber);
+            return new OptimizedUniformAssignmentBuilder(groupSpec, subscribedTopicDescriber)
+                .build();
```

Comment:
any reason why we changed the name to not match the general assignor? Or is this also changed in the original that renamed the files?

Reply:
I thought that `assignment` was redundant as this is an assignment builder. I will also rename the general one but I will do it in a different PR.